index.js

```diff
@@ -133,9 +133,9 @@ async function send (ctx, path, opts = {}) {
     }
     ctx.set('Cache-Control', directives.join(','))
   }
-  ctx.type = type(path)
   ctx.body = fs.createReadStream(path)
-
+  ctx.type = ctx.res.type ? ctx.res.type : type(path);
```

Comment:
indeed, but

1. `ctx.response.type`, not `ctx.res.type`
2. you need add a test case for this :)